build.js

```diff
@@ -61,12 +61,7 @@ async function main() {
       outdir,
       entryPoints: filesToCompile
         .concat('./index.js')
-        .map(file => path.resolve(file)),
-
-      // minify the compiled code
-      minify: true,
-      // Keep `constructor.name` the same (used for associations)
-      keepNames: true
```

Comment:
Why was `keepNames` removed here as well?

Reply:
`keepNames` is only relevant if `minify` is present - it ensures that classes and such keep their name.